README.md

```diff
@@ -18,8 +18,7 @@ if not, file an issue.
 ### Documentation quick links
 
 * [Why octoml-profile?](#why-octoml-profile)
-* [Installation](#installation)
-* [Getting started](#getting-started)
+* [Installation and Getting Started](#installation-and-getting-started)
```

Comment:
There is a place above which references "#installation" that needs to be changed.

Reply:
thanks for catching that!